tests/test_dice_loss.py

```diff
@@ -114,13 +114,13 @@
 class TestDiceLoss(unittest.TestCase):
     @parameterized.expand(TEST_CASES)
     def test_shape(self, input_param, input_data, expected_val):
-        result = DiceLoss(**input_param).forward(**input_data)
+        result = DiceLoss(**input_param)(**input_data)
```

Comment:
this actually prevents potential bugs, thanks!

Reply:
I changed this back to have .forward